### ISSUE TITLE
Hi, is there a plan to support multiple inputs model or multimodal model?

### ISSUE THREAD

**Issue by leo-zhang** [jacobgil/pytorch-grad-cam#159]
I have a multiple inputs (like: image mask and text) model. Is there a simple way to operate various cam diagrams, or is there a plan to support multiple inputs or multimodal model?

**Comment by ashutosh-iitg** [comment]
> Hi, Currently no simple way. I have an experimental branch that might also end up supporting this (for example, defining a custom loss function).
> 
> If there is some open source use case like image/text you can point to (maybe image captioning?), this could be a good starting point to think about how to add it.

Hi @jacobgil !
Is there a way we can use the current implementation of GradCAM for a model that takes multiple images as input for a classification task (e.g.a Multi-View CNN or 3D-CNN)? 

Here's a [reference](https://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0245230) paper for MVCNN.

**Comment by jacobgil** [comment]
No way of using the code as is, it would require modifications in the code.
For example, an extra dimension would need to be added here:
https://github.com/jacobgil/pytorch-grad-cam/blob/master/pytorch_grad_cam/base_cam.py#L55

If there is some open source model/images for MVCNN, could try taking a look at adapting for it.
